src/Microsoft.TestPlatform.Extensions.HtmlLogger/TestResults.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Microsoft.VisualStudio.TestPlatform.Extensions.HtmlLogger
+{
+    [DataContract]
+    [KnownType(typeof(TestResult))]
+    public sealed class TestResults
```

Comment:
Something like TestRunDetails.
But let's discuss, this seems like a redundant class.